### PR TITLE
[O11y][Prometheus] Migrate Prometheus Server Overview dashboard to lens

### LINKED ISSUE #7776: [O11y][Prometheus] Prometheus Lens migration
Manually migrate `Prometheus` visualizations to the lens in the Kibana version `8.9.0` itself. 

## Preparation of data for testing
- [x] Use sample_event.json / setup live instance using docker / mock data using a mock server to populate dashboards 

## Migration stats
<html>
<body>

  | Before Migration |   | After Migration |  
-- | -- | -- | -- | --
  | Lens | Vizualization | Lens | Vizualization
[Metrics Prometheus] Prometheus Server overview | 0 | 12 | 12  |  0

</body>
</html>

## Dashboard : [Metrics Prometheus] Prometheus Server overview
  - [x] Migrate  visualization to the lens.
    - https://github.com/elastic/integrations/pull/8016
## Verification and Validation
  - [x] Verification of functionality remains the same after migration in Kibana
  - [x] Verification of data count of after-migrated visualization with before-migrated visualization in Kibana
  - [x] Verification of Dashboards is not distorted in the supported Kibana version ^8.9.0

### PR DESCRIPTION
**Pull request by rajvi-elastic**
### Urgency
- High

### Activity Type
- Enhancement

## What does this PR do?
- Manually migrate `Prometheus` visualizations to the lens in the current Kibana version `8.9.0` itself. 

## Migration stats
<html>
<body>

  | Before Migration |   | After Migration |  
-- | -- | -- | -- | --
  | Lens | Vizualization | Lens | Vizualization
[Metrics Prometheus] Prometheus Server overview | 0 | 12 |  12 | 0 

</body>
</html>

## Checklist

- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that panels are populated with data.
- [X] I have verified that panels are not distorted after being migrated to the lens.
- [x] I have updated screenshots of the dashboard.
- [X] I have verified that the data counts are matching and panel aggregations are the same as before.

## Author's Checklist

- [X] Migrated panels should be removed from visualization folder.
- [X] Migrated visualizations are populating in current Kibana version 8.9.0 itself.

## Related issues

- Closes #7776

### CODE REVIEW

File: packages/prometheus/kibana/dashboard/prometheus-c181a040-3d96-11ed-b624-b12467b8df74.json [hunk 1]
```diff
@@ -214,182 +336,330 @@
                 },
                 "panelIndex": "8bf9496e-83bf-404b-9465-b4497e7d3c64",
                 "title": "Appended Samples rate  [Metrics Prometheus]",
-                "type": "visualization",
-                "version": "8.4.0"
+                "type": "lens",
+                "version": "8.7.0"
```
Comment (gizas) [inline packages/prometheus/kibana/dashboard/prometheus-c181a040-3d96-11ed-b624-b12467b8df74.json:340]: Just from curiosity, is this extracted from 8.9.0 and why here we have 8.7.0?

File: packages/prometheus/kibana/dashboard/prometheus-c181a040-3d96-11ed-b624-b12467b8df74.json [hunk 2]
```diff
@@ -1330,18 +2526,19 @@
...
             }
         ],
         "timeRestore": false,
         "title": "[Metrics Prometheus] Prometheus Server overview",
         "version": 1
     },
-    "coreMigrationVersion": "8.4.0",
+    "coreMigrationVersion": "8.7.0",
```
Comment (gizas) [inline packages/prometheus/kibana/dashboard/prometheus-c181a040-3d96-11ed-b624-b12467b8df74.json:2537]: Same here this remained to 8.7.0. Is this expected?

File: packages/prometheus/kibana/dashboard/prometheus-c181a040-3d96-11ed-b624-b12467b8df74.json (hunk 1 above)
Comment (rajvi-elastic) [reply packages/prometheus/kibana/dashboard/prometheus-c181a040-3d96-11ed-b624-b12467b8df74.json:340]: @gizas, Dashboards could not be exported from 8.9.0 due to below error. Therefore I have exported dashboard from 8.7.0 as minimum Kibana version for Prometheus is 8.9.0.
image

**Review by gizas — APPROVED** [review]

**Review by kush-elastic — APPROVED** [review]
LGTM!

**Review by kush-elastic — APPROVED** [review]
LGTM!

**Review by kush-elastic — APPROVED** [review]
LGTM!

**Pull request merged**